web-marketplace/src/hooks/useMsgClient.ts

```diff
@@ -76,12 +76,15 @@ export default function useMsgClient(
 
   const sign = useCallback(
     async (tx: TxData): Promise<Uint8Array | undefined> => {
-      if (!signingClient || !wallet?.address) return;
+      const client = signingCosmWasmClient ?? signingClient;
```

Comment:
Actually, I think we might not need the regular `signingClient` anymore in `ledger.tsx`
since now both `signingCosmWasmClient` and `signingClient` register the same types
which makes `signingCosmWasmClient` just an extended `signingClient` with cosmwasm related functions